examples/eth_bots/eth_bots_config.py

```diff
@@ -0,0 +1,87 @@
+"""Experiment configuration"""
+from __future__ import annotations
+
+import logging
+
+from fixedpointmath import FixedPoint
+
+from elfpy.agents.policies import Policies
+from elfpy.bots import AgentConfig, Budget, EnvironmentConfig
+
+# You can import custom policies here. For example:
+from .custom_policies.example_custom_policy import ExampleCustomPolicy
+
+
+def get_eth_bots_config() -> tuple[EnvironmentConfig, list[AgentConfig]]:
```

Comment:
So I really like named tuples for these kindsa returns.  then I can do `eth_bots_config.environment_config`.  try it out sometime, see how it feels.